pkg/sdn/api/v1/conversion.go

```diff
@@ -2,27 +2,75 @@ package v1
 
 import (
 	kapi "k8s.io/kubernetes/pkg/api"
+	conversion "k8s.io/kubernetes/pkg/conversion"
 
 	oapi "github.com/openshift/origin/pkg/api"
-	"github.com/openshift/origin/pkg/sdn/api"
+	newer "github.com/openshift/origin/pkg/sdn/api"
+)
+
+const (
+	KnownInvalidVNID = ^uint(0)
```

Comment:
If this is part of the API, it needs to be a constant in the `types.go`, not in the conversions.  Also, it'll need to have a decimal value since that's how the API users will see it.